### PR TITLE
[fix][adascale] Fix infinite loop in docstring

### PR DESCRIPTION
**Pull request by anj-s**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [x] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?
- [ ] Did you update the [changelog](https://github.com/facebookresearch/fairscale/blob/master/CHANGELOG.md)? (if needed)

## What does this PR do?
Fixed infinite loop in docstring with other minor changes.

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by min-xu-ai — COMMENTED** [review]
Thanks! Rename done to not_done?

File: fairscale/optim/adascale.py [hunk 1]
```diff
@@ -72,9 +72,9 @@ class AdaScale(Optimizer):
         scheduler = LambdaLR(optim, lr_lambda=...)
 
         last_epoch = 0
-        done = False
+        done = True
```
Comment (min-xu-ai) [inline fairscale/optim/adascale.py:75]:
```suggestion
        done = False
```

File: fairscale/optim/adascale.py [hunk 2]
```diff
@@ -72,9 +72,9 @@ class AdaScale(Optimizer):
...
         step = 0
-        while True:
+        while done:
```
Comment (min-xu-ai) [inline fairscale/optim/adascale.py:77]:
```suggestion
        while not done:
```

File: fairscale/optim/adascale.py [hunk 3]
```diff
@@ -86,8 +86,8 @@ class AdaScale(Optimizer):
                 if epoch > last_epoch:
                     scheduler.step()
                     last_epoch = epoch
-                if epoch >= max_epochs:
-                    done = True
+                if epoch >= MAX_EPOCHS:
+                    done = False
```
Comment (min-xu-ai) [inline fairscale/optim/adascale.py:90]:
```suggestion
                    done = True
```

**Review by min-xu-ai — APPROVED** [review]
I was thinking that we can just change the while True to while not done to fix this.

File: fairscale/optim/adascale.py (hunk 1 above)
Comment (anj-s) [reply fairscale/optim/adascale.py]: Yes, this improves it. I also checked the tutorial and that is how the loop is written there as wel.

**Pull request merged**